### DISCUSSION TITLE
Window focus control based on arrow keys

### DISCUSSION THREAD

**Discussion by AbdallahAlhaddad** [rxhanson/Rectangle#1312] [Ideas]
I think it would be more intuitive if we can change current focused window based on arrow keys as implemented in [Pop!_OS](https://www.youtube.com/watch?v=-fltwBKsMY0) linux distro.

**Comment by rxhanson** [comment]
Thanks for the feedback!

I don't have plans to add this in myself, but would welcome a good pull request for it. All new additional features to Rectangle are restricted to being enabled via terminal command, and are only added to the UI if there is enough interest in the feature.

**Reply by petervines** [reply]
I would second this idea. It would be great for example if you had windows arranged in quarters or sixths to be able to switch between them quickly, forward and back with the keyboard. As it is you have to use command-tab which shows apps, but with no relationship to the screen grid of windows. Alternatively you use the mouse which I would like to avoid. There is a standard Mac shortcut opt-F4 which switches to the next window, but this is not restricted to one display. Also you can only go forward to the next window, this is probably the best option right now.
